Update http-requests-exchange-rates.md (#909)
dfinity/portal: docs/samples/http-requests-exchange-rates.md

## Exchange Rate sample dapp overview

> The [Canister HTTPS outcalls](https://wiki.internetcomputer.org/wiki/HTTPS_outcalls) feature provides a way for canisters to directly interact with applications and data that exist outside of Internet Computer in the Web 2.0 world. Our Exchange Rate sample dapp is created to demonstrate usage of the new Canister HTTP feature. You provide implementations in [Rust](https://github.com/dfinity/examples/tree/master/rust/exchange_rate) and [Motoko](https://github.com/dfinity/examples/tree/master/motoko/exchange_rate).

Reviewer: ```suggestion
The [Canister HTTPS outcalls](https://wiki.internetcomputer.org/wiki/HTTPS_outcalls) feature provides a way for canisters to directly interact with applications and data that exist outside of Internet Computer in the Web 2.0 world. Our Exchange Rate sample dapp is created to demonstrate usage of the new Canister HTTP feature. Here are implementations in [Rust](https://github.com/dfinity/examples/tree/master/rust/exchange_rate) and [Motoko](https://github.com/dfinity/examples/tree/master/motoko/exchange_rate).
```
